docs/preview/02-Features/02-message-handling/01-service-bus.md

```diff
@@ -398,8 +380,124 @@ public class Startup
 }
 ```
 
-## Alternative Service Bus message routing
+## Pump Configuration
+Next to that, we provide a **variety of overloads** to allow you to:
+- Specify the name of the queue/topic
+- Only provide a prefix for the topic subscription, so each topic message pump is handling messages on separate subscriptions
+- Configure how the message pump should work *(ie. max concurrent calls & auto delete)*
+- Read the connection string from the configuration *(although we highly recommend using the [Arcus secret store](https://security.arcus-azure.net/features/secret-store) instead)*
```

Comment:
Maybe [this](https://docs.microsoft.com/en-us/azure/app-service/app-service-key-vault-references#reference-syntax) syntax is supported.

Reply:
How would this help? This is about the Arcus secret store, right?

Reply:
Ah, that's cool, but maybe we should still refer to our secret store to make the difference clear between configuration data <> secrets. Could be further investigated, of course.

Reply:
Great!